TYPO3.Neos/Documentation/CreatingASite/RenderingCustomMarkup/CustomContentElements.rst

```diff
@@ -263,6 +263,7 @@ Registering 2 new groups could look like::
 	        special:
 	          position: 50
 	          label: 'Special elements'
+	          collapsed: true
```

Comment:
since the groups aren't actually documented anywhere I added it here, since it's the only place it's mentioned.. tried to find a good place to document the group configuration in general, but didn't find a suiting place..